### PR TITLE
fix(multi_object_tracker): fix reversed width and length in bounding box

### PR DESCRIPTION
**Pull request by YoshiRi**
## Description

Solved the problem of width and length in member variable `bonding_box_` being reversed.

- Fix the description in which `bounding_box_.width` and `object.shape.dimensions.x`(representing length) was associated with each other.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp [hunk 1]
```diff
@@ -130,7 +130,7 @@ BicycleTracker::BicycleTracker(
 
   if (object.shape.type == autoware_auto_perception_msgs::msg::Shape::BOUNDING_BOX) {
     bounding_box_ = {
-      object.shape.dimensions.x, object.shape.dimensions.y, object.shape.dimensions.z};
+      object.shape.dimensions.y, object.shape.dimensions.x, object.shape.dimensions.z};
```
Comment (yukkysaito) [inline perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp:133]: It is better to change the order of variable at struct definition.

File: perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp [hunk 2]
```diff
@@ -132,7 +132,7 @@ BicycleTracker::BicycleTracker(
     bounding_box_ = {
       object.shape.dimensions.x, object.shape.dimensions.y, object.shape.dimensions.z};
   } else {
-    bounding_box_ = {1.0, 0.5, 1.7};
```
Comment (YoshiRi) [inline perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp:135]: I am concerning if this default bicycle shape is correct or not. 
I think longitude side should longer in normal bicycle.

File: perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp (hunk 1 above)
Comment (YoshiRi) [reply perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp]: That's reasonable. I updated the PR.

**Comment by YoshiRi** [comment]
closed because included in https://github.com/autowarefoundation/autoware.universe/pull/2238

**Pull request closed without merging**